packages/functions/src/coordinator/GetCoordinatorHandler.ts

```diff
@@ -11,11 +12,15 @@ export default async function (
     console.error("Could not find calling user", callerId);
     throw Error(`User is not a coordinator`);
   }
+  const coordinatorDonorUser = await DonorDAL.getDonor(callerId);
```

Comment:
Please run in parallel.

This means to call `DonorDAL.getDonor(callerId);` before you call `getCoordinator` but without the `await`, and only `await` on it later

Reply:
I see you marked as resolved, but I don't see any code changes